build/bin/sage-guess-package-system

```diff
@@ -30,12 +30,5 @@ elif xbps-install --version > /dev/null 2>&1; then
 elif pkg -v > /dev/null 2>&1; then
     echo freebsd
 else
-    case `uname -s` in
-        CYGWIN*)
```

Comment:
All other removals and changes LGTM.

Reply:
> what's the point of keeping cygwin.txt ?

Because we have a nice little information system there, which I want to offer to other projects via #31662.

Reply:
> I don't know but there are 100 other things needed to remove cygwin so I don't really care if we save that for last. Feel free to revert that commit or rebase it away.

OK, I'll split this part out to a separate PR that I'll mark "pending".

Reply:
That's now #36782.

Reply:
> > what's the point of keeping cygwin.txt ?
> 
> Because we have a nice little information system there, which I want to offer to other projects via #31662.

Cygwin was dragging down Sage -  are you keen on offering this burden to other projects? :-)